secretmanager/src/test/java/secretmanager/SnippetsIT.java

```diff
@@ -154,6 +160,7 @@ private static Secret createSecret() throws IOException {
                             .setAutomatic(Replication.Automatic.newBuilder().build())
                             .build())
                     .putLabels(LABEL_KEY, LABEL_VALUE)
+                    .putAnnotations(ANNOTATION_KEY, ANNOTATION_VALUE)
```

Comment:
This will add the annotation to multiple secrets that call this method. Instead wydt about adding it only for TEST_SECRET?

Reply:
Fixed